demos/keyboard_nav/index.html

```diff
@@ -13,13 +13,38 @@
       background-color: #fff;
       font-family: sans-serif;
     }
+
     h1 {
       font-weight: normal;
       font-size: 140%;
     }
-
-    .blockRenderDebug {
-      display: none;
+    .wrapper {
+      display: flex;
+    }
+    #keyboard_nav {
+      border: 1px solid black;
+      background-color: #ededed;
```

Comment:
nit: reorder blackground-color before border to keep css alphabetical. similar comment for L44.

Reply:
done